server/controllers/user-controller.js

```diff
@@ -0,0 +1,80 @@
+import {
+  successResponse,
+  errorResponse,
+  validate,
+  validationErrorResponse,
+  checkUniqueUserName
+} from '../utils/helpers';
+import db from '../models';
+import { profileSchema } from '../utils/validators';
+
+const { User } = db;
+
+/**
+ * The controllers for users route
+ *
+ * @class UsersController
+ */
+class UsersController {
+  /**
+   * update user profile
+   * @static
+   * @param {Request} req request object
+   * @param {Response} res response object
+   * @memberof {Users}
+   * @returns {undefined}
+   */
+  static async updateUser(req, res) {
+    validate(req.body, profileSchema)
+      .then(async () => {
+        const { ...auth } = req.authUser;
+        try {
+          const { username, bio } = req.body;
```

Comment:
Also I see you're no longer handling image uploads

Reply:
handling image upload from the backend is not efficient from my research. so I will be handling it from the frontend

Reply:
I see. In that case why is cloudinary still a dependency here?

Reply:
Still seeing cloudinary as a dependency